stable/aad-pod-identity/README.md

```diff
@@ -0,0 +1,132 @@
+# aad-pod-identity
+
+[aad-pod-identity](https://github.com/Azure/aad-pod-identity) enables Kubernetes applications to access cloud resources securely with [Azure Active Directory](https://azure.microsoft.com/en-us/services/active-directory/) (AAD).
+
+## TL;DR:
+
+```console
+$ helm install stable/aad-pod-identity -f values.yaml
+```
+
+## Introduction
+
+A simple [helm](https://helm.sh/) chart for setting up the components needed to use [Azure Active Directory Pod Identity](https://github.com/Azure/aad-pod-identity) in Kubernetes.
+
+This helm chart will deploy the following resources:
+* AzureIdentity `CustomResourceDefinition`
+* AzureIdentityBinding `CustomResourceDefinition`
+* AzureAssignedIdentity `CustomResourceDefinition`
+* AzurePodIdentityException `CustomResourceDefinition`
+* AzureIdentity instance (optional)
+* AzureIdentityBinding instance (optional)
+* Managed Identity Controller (MIC) `Deployment`
+* Node Managed Identity (NMI) `DaemonSet`
+
+## Getting Started
+The following steps will help you create a new Azure identity ([Managed Service Identity](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview) or [Service Principal](https://docs.microsoft.com/en-us/azure/active-directory/develop/app-objects-and-service-principals)) and assign it to pods running in your Kubernetes cluster.
+
+### Prerequisites
+* [Azure Subscription](https://azure.microsoft.com/)
+* [Azure Kubernetes Service (AKS)](https://azure.microsoft.com/services/kubernetes-service/) or [AKS Engine](https://github.com/Azure/aks-engine) deployment
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) (authenticated to your Kubernetes cluster)
+* [Helm v1.10+](https://github.com/helm/helm)
+* [Azure CLI 2.0](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+* [git](https://git-scm.com/downloads)
+
+<details>
+<summary><strong>[Optional] Creating user identity</strong></summary>
+
+1. Create a new [Azure User Identity](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview) using the Azure CLI:
+> __NOTE:__ It's simpler to use the same resource group as your Kubernetes nodes are deployed in. For AKS this is the MC_* resource group. If you can't use the same resource group, you'll need to grant the Kubernetes cluster's service principal the "Managed Identity Operator" role.
+```shell
+az identity create -g <resource-group> -n <id-name>
+```
+
+2. Assign your newly created identity the role of _Reader_ for the resource group:
```

Comment:
This is only required in case of demo application. Can you please clarify that here or remove it since it does not apply to regular use.

Reply:
Done